### PR TITLE
Add carthage instal steps

### PR DESCRIPTION
**Pull request by mrtnrst**
## Changes in this pull request
Added Carthage Install Steps #13 

## Pull request checklist
- [x] All tests pass. Demo project builds and runs.
- [x] I added tests, an experiment, or detailed why my change isn't tested.
- [x] I have reviewed the [contributing guide](https://github.com/Instagram/IGListKit/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +24,24 @@ A data-driven `UICollectionView` framework for building fast and flexible lists.
...
+### Cocoapods
 The preferred installation method for `IGListKit` is with [CocoaPods](http://cocoapods.org). Simply add the following to your Podfile:
 
 ```ruby
 # Latest release of IGListKit
 pod 'IGListKit'
 ```
 
+### Carthage
+[Carthage](https://github.com/Carthage/Carthage) is a decentralized dependency manager that builds your dependencies and provides you with binary frameworks.
```
Comment (jessesquires) [inline README.md]: let's remove this line, and move the Carthage link to the "Carthage" text below 😄

File: README.md [hunk 2]
```diff
@@ -24,13 +24,24 @@ A data-driven `UICollectionView` framework for building fast and flexible lists.
...
+
+To integrate IGListKit into your Xcode project using Carthage, specify it in your `Cartfile`:
```
Comment (jessesquires) [inline README.md]: put IGListKit in `` 😄 

> `IGListKit`

Comment (mrtnrst) [inline README.md]: On it!

**Comment by jessesquires** [comment]
Thanks @mrtnrst ! 👍  🎉

**Pull request closed without merging**